homeassistant/components/iskra/__init__.py

```diff
@@ -20,11 +21,23 @@
     Platform,
 )
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
 from homeassistant.helpers import device_registry as dr
 
 from .const import DOMAIN, MANUFACTURER
 from .coordinator import IskraDataUpdateCoordinator
 
+
+@dataclass
+class IskraRuntimeData:
+    """IskraRuntimeData class represents the runtime data for the Iskra component.
+
+    It holds a list of coordinators.
+    """
+
+    coordinators: list[IskraDataUpdateCoordinator]
```

Comment:
The `IskraConfigEntry` is not updated. Also, if you're only storing one item in the runtime data, might as well not wrap it in a dataclass

Reply:
changed, i hope i fixed it correctly